infra/wca_on_rails/production/variables.tf

```diff
@@ -48,6 +48,17 @@ variable "region" {
   description = "The AWS Region"
 }
 
+variable "rails_start_up_time" {
+  type = number
+  description = "The Startup time of the Ruby on Rails Application"
+}
+
+variable "rds_iam_identifier" {
+  type = string
+  description = "The identifier of the RDS Instance used for IAM Auth"
+  default = "db-VFBCC2563NK74KYKEYEC32YXHA"
```

Comment:
Could this potentially (definitely not necessary as a part of this PR) be abstracted away by importing the RDS into the Terraform state, rather than hard-coding the ID as a "magical" constant?